src/code/z_olib.c

```diff
@@ -254,19 +254,19 @@ void OLib_Vec3fDiff(PosRot* a, Vec3f* b, Vec3f* dest, s16 mode) {
     VecGeo geo;
 
     switch (mode) {
-        case OLIB_ADD_OFFSET:
+        case OLIB_DIFF_OFFSET:
             OLib_Vec3fDiffToVecGeo(&geo, &a->pos, b);
             geo.yaw -= a->rot.y;
             OLib_VecGeoToVec3f(dest, &geo);
             break;
 
-        case OLIB_ADD:
+        case OLIB_DIFF:
             dest->x = b->x - a->pos.x;
             dest->y = b->y - a->pos.y;
             dest->z = b->z - a->pos.z;
             break;
 
-        default: // OLIB_COPY
+        default: // OLIB_SUB_COPY
```

Comment:
Should this be `OLIB_DIFF_COPY`? Don't see an `OLIB_SUB_COPY` anywhere, but maybe I just missed it.

Reply:
Oh oops, yeah was going back and forth between DIFF/SUB for this other enum, decided to keep it consistent with the function name